### PR TITLE
hackneyed: 0.8.2 -> 0.9.1

### PR DESCRIPTION
**Pull request by somasis**
## Description of changes

https://gitlab.com/Enthymeme/hackneyed-x11-cursors/-/tags/0.9.1

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by FliegendeWurst — APPROVED** [review]

File: pkgs/data/icons/hackneyed/default.nix [hunk 1]
```diff
@@ -1,27 +1,30 @@
...
   pname = "hackneyed";
-  version = "0.8.2";
+  version = "0.9";
 
   src = fetchFromGitLab {
     owner = "Enthymeme";
     repo = "hackneyed-x11-cursors";
     rev = version;
-    sha256 = "sha256-Wtrw/EzxCj4cAyfdBp0OJE4+c6FouW7+b6nFTLxdXNY=";
+    hash = "sha256-HKX3r5vsrMnzKhV3MkjeRUB93W64orH2BZTtXX6FLTQ=";
```
Comment (kirillrdy) [inline pkgs/data/icons/hackneyed/default.nix:11]:
```
error: hash mismatch in fixed-output derivation '/nix/store/7dl055fdccghlf8wp1dnj9qqkvwasmjx-source.drv':
         specified: sha256-HKX3r5vsrMnzKhV3MkjeRUB93W64orH2BZTtXX6FLTQ=
            got:    sha256-zkkR5CywZaMWLd8w2a/E5A6RNXK+PAQYM6y55tpjOaY=

```

File: pkgs/data/icons/hackneyed/default.nix [hunk 2]
```diff
@@ -1,27 +1,30 @@
-{ lib, fetchzip, stdenvNoCC, fetchFromGitLab, xcursorgen, imagemagick6, inkscape }:
+{ lib, stdenvNoCC, fetchFromGitLab, fetchpatch, imagemagick, inkscape, xcursorgen }:
```
Comment (kirillrdy) [inline pkgs/data/icons/hackneyed/default.nix:1]: fetchpatch is unused

**Review by kirillrdy — APPROVED** [review]

File: pkgs/data/icons/hackneyed/default.nix [hunk 3]
```diff
@@ -1,27 +1,30 @@
...
-    substituteInPlace make-png.sh \
-      --replace /usr/bin/inkscape ${inkscape}/bin/inkscape
   '';
 
   enableParallelBuilding = true;
 
-  makeFlags = [ "PREFIX=$(out)" ];
+  makeFlags = [
+    "PREFIX=$(out)"
+    "INKSCAPE=${inkscape}/bin/inkscape"
```
Comment (NickCao) [inline pkgs/data/icons/hackneyed/default.nix:24]:
```suggestion
    "INKSCAPE=inkscape"
```

File: pkgs/data/icons/hackneyed/default.nix [hunk 4]
```diff
@@ -1,27 +1,30 @@
...
+    "INSTALL=install"
+    "XCURSORGEN=${xcursorgen}/bin/xcursorgen"
```
Comment (NickCao) [inline pkgs/data/icons/hackneyed/default.nix:26]:
```suggestion
    "XCURSORGEN=xcursorgen"
```

Comment (NickCao) [reply pkgs/data/icons/hackneyed/default.nix:26]: Same here.

File: pkgs/data/icons/hackneyed/default.nix (hunk 3 above)
Comment (NickCao) [reply pkgs/data/icons/hackneyed/default.nix:24]: Otherwise it breaks cross compilation.

**Review by NickCao — COMMENTED** [review]
And there is a newer version 0.9.1: https://gitlab.com/Enthymeme/hackneyed-x11-cursors/-/tags/0.9.1

**Comment by somasis** [comment]
Should build fine now. Apologizes for the spam.

**Comment by NickCao** [comment]
Build is failing on ofborg due to hash mismatch, please check.

File: pkgs/data/icons/hackneyed/default.nix [hunk 5]
```diff
@@ -1,29 +1,41 @@
...
   pname = "hackneyed";
-  version = "0.8.2";
+  version = "0.9.1";
 
   src = fetchFromGitLab {
     owner = "Enthymeme";
     repo = "hackneyed-x11-cursors";
     rev = version;
-    sha256 = "sha256-Wtrw/EzxCj4cAyfdBp0OJE4+c6FouW7+b6nFTLxdXNY=";
+    hash = "sha256-HKX3r5vsrMnzKhV3MkjeRUB93W64orH2BZTtXX6FLTQ=";
```
Comment (FliegendeWurst) [inline pkgs/data/icons/hackneyed/default.nix:11]:
```suggestion
    hash = "sha256-+7QtHgBuhJtQejiHeZ+QoedJo24LqSY51XRVLv9Ho2g=";
```

Comment (somasis) [reply pkgs/data/icons/hackneyed/default.nix]: thanks. I'm not sure why my machine keeps fetching a different hash.

**Review by FliegendeWurst — APPROVED** [review]
Result of `nixpkgs-review pr 257545` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>hackneyed</li>
  </ul>
</details>

**Review by NickCao — APPROVED** [review]

**Pull request merged**